src/server.js

```diff
@@ -40,11 +43,24 @@ function calculateReward(value, volume) {
     return "Doesn't even fit in an Orca!";
   };
 
+  // If someone pastes in units of millions of ISK, convert to units of ones of ISK.
+  // 20.25 -> 20,250,000
+  if (value < 100000) {
+    value = value * 1000000;
+  }
+  
+  value  = Math.round(value);
+  volume = Math.round(volume);
+  
+  var volume_units = Math.ceil(volume / VOLUME_UNIT);
+  var value_units  = Math.ceil(value  /  VALUE_UNIT);
+  var billed_units = Math.max(volume_units, value_units, MIN_UNITS);
+  var reward = billed_units * UNIT_COST;
```

Comment:
I see that this formula matches the documentation on the wiki, but *that* doesn't match what I was given by Jacob.  I don't really have an opinion on the *right* formula, but check with Jacob.